requirements.txt

```diff
@@ -7,3 +7,4 @@ pytz==2016.4
 requests[security]==2.10.0
 six==1.10.0
 wsgiref==0.1.2
+docker==2.1.0
```

Comment:
We have per-plugin requirements files that can be installed as extras, so this should go in plugins/docker/requirements.txt.